server.go

```diff
@@ -0,0 +1,15 @@
+package tf
+
+import (
+	"github.com/hashicorp/terraform-plugin-go/tfprotov6"
+	// "github.com/hashicorp/terraform-plugin-go/tfprotov6/server"
+)
+
+// make a tfprotov6.ProviderServer
+// and ResourceServer
+
+type ResourceServer map[string]tfprotov6.ResourceServer
```

Comment:
I'm curious what prompted including this in this PR.

Reply:
Hubris (thinking it would be quick to scaffold everything needed to build a provider) followed by accident (committing this file)